packages/api/bin/serveUtils.js

```diff
@@ -82,12 +193,57 @@ async function addReconciliationReports(reconciliationReports) {
   );
 }
 
+async function addGranulesExecutions(granulesExecutions) {
+  const knex = await getKnexClient({
+    env: {
+      ...envParams,
+      ...localStackConnectionEnv,
+    },
+  });
+
+  return await Promise.all(
+    granulesExecutions.map(async (ge) => {
+      // Fetch the Collection ID
+      const [name, version] = ge.granule.collectionId.split('___');
+      const collectionPgModel = new CollectionPgModel();
+      const collectionCumulusId = await collectionPgModel.getRecordCumulusId(knex, {
+        name: name,
+        version: version,
+      });
+
+      // Fetch the Granule ID
+      const granulePgModel = new GranulePgModel();
+      const granuleCumulusId = await granulePgModel.getRecordCumulusId(knex, {
+        granule_id: ge.granule.granuleId,
+        collection_cumulus_id: collectionCumulusId,
+      });
+
+      // Fetch the Execution ID
+      const executionPgModel = new ExecutionPgModel();
+      const executionCumulusId = await executionPgModel.getRecordCumulusId(knex, {
+        arn: ge.execution.arn,
+      });
+
+      // Create and insert the GranuleExecution record into the DB
+      const granulesExecutionsPgModel = new GranulesExecutionsPgModel();
+      const dbRecord = {
+        granule_cumulus_id: granuleCumulusId,
+        execution_cumulus_id: executionCumulusId,
+      };
+
+      await granulesExecutionsPgModel.create(knex, dbRecord);
+    })
+  );
+}
+
```

Comment:
`granules_executions` is also a little different in that it required a lookup of the granule and execution Ids prior to record creation. Also, since there is not equivalent dynamoDb table it is Postgres only.